sections/predictive-search.liquid

```diff
@@ -15,7 +15,7 @@
       {%- if predictive_search.resources.queries.size > 0 or predictive_search.resources.collections.size > 0 -%}
         <div>
           <h2 id="predictive-search-queries" class="predictive-search__heading text-body caption-with-letter-spacing">
-            {{ 'templates.search.suggestions' | t }}
+            {{- 'templates.search.suggestions' | t -}}
```

Comment:
While reviewing the output, I noticed that the whitespaces were being rendered in the inspector. This doesn't have an impact on the UI really, but it makes the output more inline with what we want.